content/_data/authors/koushartasneem.adoc

```diff
@@ -0,0 +1,11 @@
+---
+name: Tasneem Koushar
+github: koushartasneem
+---
+
+I am Tasneem Koushar and I have worked at Chromium under Google Summer of Code 2021 and Successfully created automated documentation of Puppeteer. 
+
+I want to help contributors at Jenkins and am interested in becoming one of the mentors for the project: https://www.jenkins.io/projects/gsoc/2022/project-ideas/pipeline-step-documentation-generator/
+
+My Picture:  
```

Comment:
Your profile picture (avatar) is best placed in `content/images/avatars`. Just add it as `koushartasneem.jpg` (using the same key system as for the jenkins.io author).

Note that you are able to preview how a `jenkins.io` PR is rendered by accessing the preview deployment: 
https://deploy-preview-4998--jenkins-io-site-pr.netlify.app/projects/gsoc/2022/project-ideas/

And in particular your bio at https://deploy-preview-4998--jenkins-io-site-pr.netlify.app/blog/authors/koushartasneem/.

Reply:
(note that as I merged the PR, the preview environment will disappear)